gui/network/models.py

```diff
@@ -62,6 +62,15 @@ class GlobalConfiguration(Model):
         blank=True,
         null=True,
     )
+    gc_hostname_virtual = models.CharField(
+        max_length=120,
+        verbose_name=_("Hostname (Virtual)"),
+        validators=[RegexValidator(
+            regex=r'^[a-zA-Z\.\-\_0-9]+$',
```

Comment:
I guess that regexp is kinda works, but strictly speaking it should be more like ^[A-Za-z0-9]([A-Za-z0-9\-\_\.]*[A-Za-z0-9])*$

Reply:
There is again ordering issue with the migration, I guess cause in 11.0 and 11.1 there are different sets of migrations:
```
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/graph.py", line 104, in raise_error
    raise NodeNotFoundError(self.error_message, self.key, origin=self.origin)
django.db.migrations.exceptions.NodeNotFoundError: Migration network.0007_globalconfiguration_gc_hostname_virtual dependencies reference nonexistent parent node ('network', '0006_globalconfiguration_gc_domains')
```

Reply:
I hope at some point all those fields would get tooltips.

We don't restrict hostnames here to any specific format and explicitly allow "." in the name in `validators`, so what are the chances that people won't put FQDN there?